model/config.go

```diff
@@ -2212,8 +2212,8 @@ type SamlSettings struct {
 	// User Mapping
 	IdAttribute          *string `access:"authentication"`
 	GuestAttribute       *string `access:"authentication"`
-	EnableAdminAttribute *bool   `access:"authentication"`
-	AdminAttribute       *string `access:"authentication"`
```

Comment:
Just want to ask since it's being highlighted in this PR but what does this access label do for these settings? Any documentation I could read more about since I was always unsure about it.

Reply:
@hahmadia there's some Godoc related to it: https://github.com/mattermost/mattermost-server/blob/2bf6992397b50f2c5a7cf84480daed39a5b4b0fa/model/config.go#L2829-L2859